model/Index/Service/ResultIndexer.php

```diff
@@ -27,16 +27,14 @@
 use oat\tao\model\taskQueue\QueueDispatcherInterface;
 use oat\taoAdvancedSearch\model\Index\Normalizer\NormalizerInterface;
 
-class ResultIndexer extends ConfigurableService implements IndexerInterface
+class ResultIndexer extends ConfigurableService implements IndexerInterface, NormalizerAwareInterface
 {
     /** @var NormalizerInterface */
     private $normalizer;
 
-    public function setNormalizer(NormalizerInterface $normalizer): self
+    public function setNormalizer(NormalizerInterface $normalizer): void
```

Comment:
Are you sure this will not break anything?

Reply:
It broke, then I fixed all the occurrences :)